README.md

```diff
@@ -115,48 +88,16 @@ Postgres ssl mode is enabled by default. To disable it please change the followi
   ```
   configured in the main.yml
```

Comment:
Same - is it <trustification-ansible.git>/vars/main.yml?

Reply:
Added the entire path